### PR TITLE
add missing deps marked and prism

### PR DESCRIPTION
**Pull request by e111077**
Fixes marked-element tests. ~Something else is also messing with prism-element~ fixed.

### CODE REVIEW

File: dependency-map.json [hunk 1]
```diff
@@ -231,10 +231,18 @@
...
+  "marked": {
+    "npm": "marked",
+    "semver": "~0.3.9"
+  },
   "marked-element": {
     "npm": "@polymer/marked-element",
     "semver": "^3.0.0-pre.8"
   },
+  "prism": {
+    "npm": "prism",
```
Comment (justinfagnani) [inline dependency-map.json]: shouldn't this be `prismjs`?

**Comment by stramel** [comment]
I'm confused... isn't `marked` not setup to be a module or does that matter?

**Comment by justinfagnani** [comment]
@stramel it might not matter. Some scripts actually work when loaded as modules. They still write to globals, and references to them won't be rewritten by modulizer. I've looked at Prism and I think it'll work. We'll see with marked.

File: dependency-map.json (hunk 1 above)
Comment (e111077) [reply dependency-map.json]: Yes, yes it should

**Comment by e111077** [comment]
@justinfagnani is right, the import just seems to load it as a script. Marked seems to work as well no problem

**Comment by e111077** [comment]
That seemed to do the trick to fix prism-element. I'm impressed at how resilient the modulizer is already.

**Comment by e111077** [comment]
investigating travis failure

**Comment by justinfagnani** [comment]
@e111077 awesome!

You need to update the integration tests, which are expecting marked to not be found.

**Pull request merged**